pkgs/development/python-modules/yubico-client/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, lib, buildPythonPackage, isPyPy, fetchFromGitHub
+, requests, python-dateutil, pytz }:
+
+buildPythonPackage rec {
+  pname = "yubico-client";
+  version = "1.10.0";
+
+  disabled = isPyPy;
```

Comment:
According to https://github.com/Kami/python-yubico-client/blob/master/tox.ini#L2, this is explicitly tested with PyPy.